app/controllers/games_controller.rb

```diff
@@ -0,0 +1,38 @@
+class GamesController < ApplicationController
+  before_action :authenticate_user!, only: [:new, :create, :edit, :update, :destroy]
```

Comment:
Optional: There are no `edit` or `destroy` actions in the controller, so they don't need to be mentioned on this line.